docs/create-your-own-dapp.md

```diff
@@ -43,9 +48,11 @@ $ yarn add @nosplatform/api-functions
 
 ## Developing with nOS API
 
-Assuming you run your webapp at `http://localhost:8080`, simply type `localhost:8080` into the nOS
+Assuming you run your webapp at `http://localhost:8080`, simply type `nos://localhost:8080` into the nOS
 address bar (which you built/compiled in the pre-requisites step earlier) at the top of that application, then press <kbd>Enter</kbd>.  Refer to the
 [API documentation](./api.md) for a complete list of functionality.
 
-In the near future, an `nos-mock` library will be introduced for developing locally without needing
-the nOS client to run.
+## Known issues
+Combining a custom protocol like `nos://` with hot module replacement is a known issue, causing the nOS client to return a black screen when running your dApp.
+In such cases, it is adviced to use a bundler without hot module replacement (f.e. in Parcel this is done by adding `--no-hmr` to your start command.
+If you really want to use HMR, you can use `http://` within the client instead (though we would advice against it).
```

Comment:
Because there's no whitespace between these lines, they're showing up as a single paragraph.  I'd suggest adding a line break between each one, or making them bullet points.